releng/org.eclipse.xtext.dev-bom/pom.xml

```diff
@@ -107,6 +107,7 @@
 		<osgi-version>3.13.200</osgi-version>
 		<text-version>3.8.0</text-version>
 		<xpand-version>2.0.0</xpand-version>
+		<xtext-antlr-generator-version>2.1.1</xtext-antlr-generator-version>
```

Comment:
is this the latest available one?

Reply:
why is this used from gradle only?

Reply:
> is this the latest available one?

2.1.1 is the latest version
https://search.maven.org/search?q=g:org.eclipse.xtext%20AND%20a:xtext-antlr-generator&core=gav

> why is this used from gradle only?
Not sure, but that's how it was already. Maybe it is also not required for Gradle? I'll check.

Reply:
maybe the download way is discouraged from gradle

Reply:
I checked with Maven. If we add this, `http://download.itemis.com/antlr-generator-3.2.0-patch.jar` is not downloaded anymore. This actually sounds desirable, i.e. I would prefer to add this dependency to the runtime module. 
I'll update the PR.

Reply:
anything left to do here?